openssl/Cargo.toml

```diff
@@ -25,6 +25,7 @@ bitflags = "1.0"
 cfg-if = "1.0"
 foreign-types = "0.3.1"
 libc = "0.2"
+num_enum = "0.5"
```

Comment:
not used anywhere, is it?

Reply:
Right, not anymore. I had enums before, but Steven preferred constants.